src/components/diagrams/singleLineDiagram/single-line-diagram-pane.js

```diff
@@ -318,6 +321,37 @@ export function SingleLineDiagramPane({
 
     const displayedIds = new Set(displayedSLD.map(({ id }) => id));
     const minimized = views.filter(({ id }) => !displayedIds.has(id));
+    const [computedHeight, setComputedHeight] = useState();
+
+    useEffect(() => {
+        let displayedSldHeights_ = displayedSldHeightsRef.current;
+        displayedSldHeights_ = displayedSldHeights_?.filter((displayedHeight) =>
+            viewsRef.current
```

Comment:
use views here as you have it in the dependencies

Reply:
Done